django/thunderstore/repository/cache.py

```diff
@@ -21,19 +20,17 @@ def order_package_listing_queryset(
     queryset: QuerySet[PackageListing],
 ) -> QuerySet[PackageListing]:
     return queryset.order_by(
-        "-package__is_pinned", "package__is_deprecated", "-package__date_updated"
+        "-package__is_pinned",
+        "package__is_deprecated",
+        "-package__date_updated",
     )
 
 
 def get_package_listing_queryset(community_identifier: str) -> QuerySet[PackageListing]:
     return order_package_listing_queryset(
         prefetch_package_listing_queryset(
-            PackageListing.objects.active()
-            .exclude(~Q(community__identifier=community_identifier))
-            .exclude(review_status=PackageListingReviewStatus.rejected)
-            .exclude(
-                Q(community__require_package_listing_approval=True)
-                & ~Q(review_status=PackageListingReviewStatus.approved)
-            )
-        )
+            PackageListing.objects.active()  # type: ignore
```

Comment:
Not a blocker, but type ignore hardly seems like the correct way to handle this? You can explicitly annotate the `PackageListing.objects` type in that class to fix it. But even if it's not addressed, it'd be better to just let it be an error instead of muting it IMO.